package.json

```diff
@@ -2,7 +2,7 @@
   "name": "panzoom",
   "version": "2.5.0",
   "description": "Pan and zoom SVG elements",
-  "main": "index.js",
+  "main": "dist/panzoom.min.js",
```

Comment:
Why this change?

Reply:
@mattdax hey man, i wonder from where can i pull the code, i pretty much a noob in coding, and i was wondering if can just pull the code like this with the fixes that you made, it would be lifesaver  <script> src='https://unpkg.com/panzoom@9.4.0/dist/panzoom.min.js' </script>